components/auth/FormField.tsx

```diff
@@ -0,0 +1,55 @@
+import Ionicons from "@expo/vector-icons/Ionicons";
+import React, { useState } from "react";
+import { View, Text, TextInput, TextInputProps, TouchableOpacity } from "react-native";
+
+interface FormInputGroups {
+  title: string;
+  value: string;
+  placeholder: string;
+  handleChangeText: (e: string) => void;
+}
+
+type CustomInputGroups = TextInputProps & FormInputGroups;
+
+const FormField = ({
+  title,
+  value,
+  placeholder,
+  handleChangeText,
+  ...props
+}: CustomInputGroups) => {
+  const [showPassword, setShowPassword] = useState(false)
+
+  return (
+    <>
+      <View className="space-y-2 mt-7">
+        <Text className="text-base text-neutral-100">{title}</Text>
+        <View className="w-full border-neutral-600 border-2 focus:border-neutral-400 justify-center h-16 my-4 rounded-2xl bg-neutral-700 flex-row">
+          <TextInput
+            value={value}
+            placeholder={placeholder}
+            placeholderTextColor={"#fafafa"}
+            onChangeText={handleChangeText}
+            className="flex-1 pl-2 text-xl text-neutral-50"
+            secureTextEntry={title === "Password" && !showPassword}
```

Comment:
When you have a situation using a text more than once in the same file, create a constant for it 
const password = "Password" above
`secureTextEntry={title === password && !showPassword}`

It helps to avoid error and if you have to change it, changing it in one place, changes everything

Reply:
Done